### PR TITLE
Add player

### PR DESCRIPTION
**Pull request by dajanaheinze**

### CODE REVIEW

File: src/components/TutorialPage.js [hunk 1]
```diff
@@ -9,17 +10,19 @@ export default function TutorialPage() {
...
+          imagefile,
+          audiofile,
           description1,
           description2,
           description3,
           id,
         }) => (
           <TutorialWrapper key={id}>
-            <img src={image} alt="" />
+            {imagefile && <img src={imagefile} alt="" />}
```
Comment (JulianBlohm) [inline src/components/TutorialPage.js:20]: In case there is no image, makes sense 👍🏻

File: src/components/AudioButton.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  }, [isSoundPlaying])
+
+  function toggleSound() {
+    setIsSoundPlaying(!isSoundPlaying)
+  }
+
+  return (
+    <Button onClick={toggleSound}>
+      {isSoundPlaying ? <PauseIcon /> : <PlayIcon />}
```
Comment (JulianBlohm) [inline src/components/AudioButton.js:37]: Nice and clean logic :)

File: src/components/TutorialPage.js [hunk 3]
```diff
@@ -33,8 +36,7 @@ const TutorialWrapper = styled.div`
   display: grid;
   border: 1px solid var(--primary-border);
   background-color: var(--secondary-background);
-  box-shadow: 20px 20px 60px var(--primary-shadow),
-    -10px -10px 50px var(--secondary-shadow);
+  box-shadow: 20px 20px 60px var(--primary-shadow);
```
Comment (JulianBlohm) [inline src/components/TutorialPage.js:38]: Like your color variables, clear naming and well prepared!

File: src/components/AudioButton.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { Howl } from 'howler'
+import { useEffect, useState, useRef } from 'react'
```
Comment (JulianBlohm) [inline src/components/AudioButton.js]: Importing React is not necessary anymore with the newest "create-react-app" version.

File: src/components/TutorialPage.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
```
Comment (JulianBlohm) [inline src/components/TutorialPage.js]: Here also: Import not necessary :)

**Review by JulianBlohm — COMMENTED** [review]
Nice new feature! 🥳
You can take out the imports of "React" in general, they are not needed anymore with the actual version of react :) 
(placed two comments about this, but its in every file I guess).
Looks very nice to me in general, keep up the good work.

File: src/components/AudioButton.stories.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+import AudioButton from './AudioButton'
+
+export default {
+  title: 'Components/Audio Button',
+  component: AudioButton,
+}
+
+const Template = (args) => <AudioButton {...args} />
+
+export const Primary = Template.bind({})
```
Comment (Hecktors) [inline src/components/AudioButton.stories.js:11]: How does that work? 😲 Crazy advanced stuff!

**Review by Hecktors — COMMENTED** [review]
Everything fine, great job. Nothing to complain. Can't wait to use your app!

File: src/components/AudioButton.stories.js (hunk 6 above)
Comment (dajanaheinze) [reply src/components/AudioButton.stories.js:11]: storybook ;-)

**Pull request merged**